app/uk/gov/hmrc/gform/summary/SummaryRenderingService.scala

```diff
@@ -366,9 +364,13 @@ object SummaryRenderingService {
       val pageTitle = page.shortName.getOrElse(page.title)
 
       val begin = source.fold { _ =>
-        begin_section(pageTitle)
+        page.presentationHint
+          .filter(_ == InvisiblePageTitle)
+          .fold(begin_section(pageTitle))(_ => HtmlFormat.empty)
       } { _ =>
-        begin_section(pageTitle)
+        page.presentationHint
+          .filter(_ == InvisiblePageTitle)
+          .fold(begin_section(pageTitle))(_ => HtmlFormat.empty)
       } { addToList =>
         addToList.presentationHint
           .filter(_ == InvisiblePageTitleInSummary)
```

Comment:
What is the difference between `InvisiblePageTitleInSummary` and `InvisiblePageTitle` ?

Reply:
`InvisiblePageTitleInSummary` was added to hide add to list title in summary page. I know the name doesnt do justice to what it does. `InvisiblePageTitle` applies to page titles, for all pages - a generic feature to disable titles